packages/sw-cli/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "sw-cli",
   "version": "0.0.8",
-  "private": true,
```

Comment:
👍 Noticed this is also in https://www.npmjs.com/package/sw-cli and we've been keeping the package updated. We might want to chat sometime about whether we version up all of the packages in the repo at the same time (e.g like Angular does) or keep them individual.